### DISCUSSION TITLE
specific domain is not blocked

### DISCUSSION THREAD

**Discussion by WebTiger89** [DNSCrypt/dnscrypt-proxy#2196] [Q&A]
Hi,
trying dnscrypt-proxy the first time on Windows 11. I have read the installation guide in the wiki and enabled _blocked-names.txt_.

content:
a1813.dscd.akamai.net
s-0005.s-msedge.net
wns.notify.trafficmanager.net
*.akamaiedge.net
*.prod.do.dsp.mp.microsoft.com

I'm using Wireshark to track network traffic and I'm wondering why _a1813.dscd.akamai.net_ is still tracked while the others are seemingly blocked:

Source Address: a1813.dscd.akamai.net (23.32.238.115)

Tbh, I'm not that familiar with network stuff. Any ideas why this specific domain is not blocked. If it's important...the domain is available under a subset of different IPs.

**Comment by lifenjoiner** [comment]
Can't reproduce what you say ...
Have you turned `blocked_names_file` on?
https://github.com/DNSCrypt/dnscrypt-proxy/blob/ca253923eb631ccde49bfcc6cf376ba867485108/dnscrypt-proxy/example-dnscrypt-proxy.toml#L519-L523
And make sure there is no rule in `allowed_names_file` covering your example.
